examples/production/values.yaml

```diff
@@ -69,6 +74,24 @@ spire-agent:
 
   logLevel: info
 
+upstream-spire-agent:
+  namespaceOverride: spire-system
```

Comment:
Again with the overrides....

Reply:
This implies an override, which implies a value that is being changed. There is little to no value in indicating that it overrides something, as Helm is a declarative tool. We should just "declare" the namespace, and not a policy of how to apply it.

Please break this out into an issue on the appropriate helm chart and don't attempt to handle this in this PR

Reply:
I'm not creating the namespaceOverride here. I'm simply using it, appropriately. Changing the name of namespaceOverride to be something else is out of scope for this PR and should be done as a separate issue.

Reply:
**Please break this out into an issue on the appropriate helm chart and don't attempt to handle this in this PR**

Reply:
I'm not creating the namespaceOverride here. I'm simply using it, appropriately. Its required for proper deployment.

Changing the name of namespaceOverride to be something else is out of scope for this PR and should be done as a separate issue.

If you would like to rename the namespaceOverride value, please file an issue for it.